tfhe/benches/integer/bench.rs

```diff
@@ -2493,6 +2614,7 @@ criterion_group!(oprf, oprf::unsigned_oprf);
 
 #[cfg(feature = "gpu")]
 fn go_through_gpu_bench_groups(val: &str) {
+    cuda_compress_ops();
```

Comment:
is this final ? i.e. run compression always ?

Reply:
I think so, unless I miss understand this method. I don't believe compression fits in "default", "fast_default" or "unchecked", so I let it out of those groups. Do you think it should go into "default"?

Reply:
could be part of default probably yes, I don't recall what the CPU stuff is separate though 🤔 @mayeul-zama ?

Reply:
or perhaps fast_default and default, as fast_default is a way to deduplicate benchmarks, and compression does not duplicate any

Reply:
Ok!
I moved it to both default and fast_default. We can change again in the future.